fenix/app/src/main/java/org/mozilla/fenix/onboarding/controller/OnboardingController.kt

```diff
@@ -28,11 +30,16 @@ interface OnboardingController {
  */
 class DefaultOnboardingController(
     private val activity: HomeActivity,
+    private val navController: NavController,
     private val hideOnboarding: () -> Unit,
 ) : OnboardingController {
 
     override fun handleStartBrowsingClicked() {
         hideOnboarding()
+
+        navController.navigate(
+            OnboardingFragmentDirections.actionGlobalHome(focusOnAddressBar = true),
+        )
```

Comment:
is there any telemetry that we have that needs to go here?

Reply:
This is a good catch. I added the telemetry event that aligned with https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/toolbar/ToolbarController.kt#77